docs/advanced/exceptions.rst

```diff
@@ -328,6 +328,28 @@ Alternately, to ignore the error, call `PyErr_Clear
 Any Python error must be thrown or cleared, or Python/pybind11 will be left in
 an invalid state.
 
+Handling warnings from the Python C API
+=====================================
+
+Wrappers for handling Python warnings are implemented in ``pybind11/warnings.h``,
+which means that ``#include`` must be added explicitly (in other words, it is not
```

Comment:
`which means` is not correct here. It was a choice to not include `warnings.h` in `pybind11.h`. We just want to state that here, to be helpful.

Could you please change this to

`which must be included explicitly`?